iwyu.cc

```diff
@@ -3799,6 +3800,20 @@ class IwyuAstConsumer
     return Base::VisitClassTemplateSpecializationDecl(decl);
   }
 
+  // Track use of namespace in using directive decl
+  // a.h:
+  //   namespace a { ... };
+  //
+  // b.cpp:
+  //   include "a.h"
+  //   using namespace a;
+  //   ...
+  bool VisitUsingDirectiveDecl(clang::UsingDirectiveDecl *decl) {
+    if (CanIgnoreCurrentASTNode())  return true;
```

Comment:
One space too many before `return`

Reply:
Actually, nevermind. I see this is almost consistently wrong. Let's merge this as-is, and I can follow up with a refactor commit to fix all lines.